Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalPedestal_cff.py

```diff
@@ -4,32 +4,68 @@
 #AlCaReco filtering for HCAL minbias:
 #------------------------------------------------
 
-from Calibration.HcalAlCaRecoProducers.ALCARECOHcalCalMinBias_cff import *
-hcalDigiAlCaMB.InputLabel = 'hltHcalCalibrationRaw'
+#from Calibration.HcalAlCaRecoProducers.ALCARECOHcalCalMinBias_cff import *
+#hcalDigiAlCaMB.InputLabel = 'hltHcalCalibrationRaw'
 #from EventFilter.HcalRawToDigi.HcalRawToDigi_cfi import *
 #hcalDigis.InputLabel = 'hltHcalCalibrationRaw'
-from RecoLocalCalo.Configuration.hcalLocalRecoNZS_cff import *
 
-hbherecoMB.firstSample = 0
-hbherecoMB.samplesToAdd = 4
-hbherecoMB.digiLabel = 'hcalDigiAlCaMB'
+import EventFilter.HcalRawToDigi.HcalRawToDigi_cfi
+hcalDigiAlCaPedestals = EventFilter.HcalRawToDigi.HcalRawToDigi_cfi.hcalDigis.clone()
+hcalDigiAlCaPedestals.InputLabel = 'hltHcalCalibrationRaw'
 
-hfrecoMB.firstSample = 0
-hfrecoMB.samplesToAdd = 2
-hfrecoMB.digiLabel = 'hcalDigiAlCaMB'
+import RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_hbhe_cfi
+hbherecoPedestals = RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_hbhe_cfi.hbheprereco.clone()
+import RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_hf_cfi
+hfrecoPedestals = RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_hf_cfi.hfreco.clone()
+import RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_hf_cfi
+horecoPedestals = RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_ho_cfi.horeco.clone()
 
-horecoMB.firstSample = 0
-horecoMB.samplesToAdd = 4
-horecoMB.digiLabel = 'hcalDigiAlCaMB'
+#from RecoLocalCalo.Configuration.hcalLocalRecoNZS_cff import *
+import RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_hbhe_cfi
+hbherecoPedestals = RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_hbhe_cfi.hbheprereco.clone()
 
+import RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_hf_cfi
+hfrecoPedestals = RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_hf_cfi.hfreco.clone()
+
+import RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_ho_cfi
+horecoPedestals = RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_ho_cfi.horeco.clone()
```

Comment:
@slava77 thanks for spotting the spurious multiple imports. Should be clean now.